src/main/scala/com.gu.contentapi.client/GuardianContentClient.scala

```diff
@@ -18,7 +13,7 @@ trait ContentApiClientLogic {
 
   val apiKey: String
 
-  implicit def executionContext = ExecutionContext.global
+  implicit def executionContext: ExecutionContext = ExecutionContext.global
```

Comment:
True. I think it's moaning because it's a `def` really. Is there any reason it's not a `val`?